src/Microsoft.TemplateSearch.Common/ISearchPackFilter.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.TemplateSearch.Common
+{
+    public interface ISearchPackFilter
```

Comment:
is this a good opportunity to comment public types & their public members?

Reply:
Added documentation